lib/executor/ramping_vus.go

```diff
@@ -54,6 +54,7 @@ func init() {
 type Stage struct {
 	Duration types.NullDuration `json:"duration"`
 	Target   null.Int           `json:"target"` // TODO: maybe rename this to endVUs? something else?
+	Progression null.String     `json:"progression"`
```

Comment:
I would prefer to name it `Interpolation` but as the naming is already being discussed in [#2256](https://github.com/grafana/k6/issues/2256), I'm fine with leaving it at that for now :+1: